templates.dist/fields.php

```diff
@@ -224,4 +224,93 @@ function renderRegionalAddition(string $inputName, array $selectedValue, bool $m
 		$output .= '</select>';
 		return $output;
 	}
-}
\ No newline at end of file
+}
+
+if (!function_exists('renderParkingLot')) {
+	function renderParkingLot(array $parkingArray, string $language, string $locale = 'de_DE'): array
+	{
+		$messages = [];
+		foreach ($parkingArray as $key => $parking) {
+			if (!$parking['Count']) {
+				continue;
+			}
+			$element = sprintf(__('%1$s at %2$s', 'onoffice'), getParkingName($key, $parking['Count']), formatPriceParking($parking['Price'], $language, $locale));
```

Comment:
`/* translators: ... */` comment is missing here

Reply:
The comment needs to explain `%1$s` and `%2$s`, which would be the _type of parking lot_ and its _price_, respectively.